src/Instrumentation/HttpAsyncClient/README.md

```diff
@@ -36,4 +36,15 @@ $tracerProvider->shutdown();
 
 ```bash
 $ composer require open-telemetry/opentelemetry-auto-http-async
-```
\ No newline at end of file
+```
+
+## Configuration
+
+Parts of this auto-instrumentation library can be configured, more options are available throught the 
```

Comment:
```suggestion
You can disable auto-instrumentation of this library by using following environment variable, more settings are available through the 
```

Reply:
In that case the table doesn't make sense, and it doesn't allow for future expansion. Either keep it like this, or we should replace it with just a "disable" section that explains the env var name